compiler/rustc_session/src/config.rs

```diff
@@ -3449,3 +3450,19 @@ impl Polonius {
         matches!(self, Polonius::Next)
     }
 }
+
+/// The different settings that the `-Zfunction-return` flag can have.
+#[derive(Clone, Copy, PartialEq, Hash, Debug)]
+pub enum FunctionReturn {
+    /// Keep the function return unmodified.
```

Comment:
use `#[default]` and derive it

Reply:
+1, done (we should change the rest in this file, by the way -- I was following `Polonius` which was recently added).